2021q4/aslr.adoc

```diff
@@ -0,0 +1,22 @@
+=== Enable ASLR by default for 64-bit executables
+
+Contact: Dawid Gorecki <dgr@semihalf.com> +
+Contact: Marcin Wojtas <mw@semihalf.com>
+
+Address Space Layout Randomization (ASLR) is an exploit mitigation
+technique implemented in the majority of modern operating systems.
+It involves randomly positioning the base address of an executable
+and the position of libraries, heap, and stack, in a process's address
+space. Although over the years ASLR proved to not guarantee full OS
+security on its own, this mechanism can make exploitation more difficult.
+
+Semihalf team made an effort to toggle the kernel settings of all 64-bit
```

Comment:
s/Semihalf/The Semihalf/

By "toggle the kernel settings...", do you mean enable the option by default"?

Reply:
Done. I rephrased the section, please check if it's more clear now.